go.mod

```diff
@@ -8,23 +8,24 @@ require (
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/google/uuid v1.1.2
-	github.com/grpc-ecosystem/grpc-gateway v1.14.6 // indirect
 	github.com/openshift/api v0.0.0-20210521075222-e273a339932a
 	github.com/openshift/build-machinery-go v0.0.0-20210423112049-9415d7ebd33e
 	github.com/openshift/client-go v0.0.0-20210521082421-73d9475a9142
 	github.com/openshift/library-go v0.0.0-20210825122301-7f0bf922c345
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.7.1
```

Comment:
why do you to update the modules for your PR?

Reply:
I think there were build error, thats why but I used the correct golang version for 4.8 to generate those

Reply:
can you revert all those please? I don't really see how updating prometheus is necessary for the changes

Reply:
running `make verify` on the branch, after resetting hard to head~2 (i.e. [commit](https://github.com/openshift/cluster-etcd-operator/pull/902/commits/1a64824e805bcce44f24953bfc53d331cf8cfdb2) ) generates the following 

```
make verify
Running `gofmt -s -l` on 80 file(s).
go vet -mod=vendor ./...
pkg/cmd/readyz/readyz.go:14:2: cannot find package "." in:
	/Users/mustafa/workspace/go/src/etcd/cluster-etcd-operator/vendor/go.etcd.io/etcd/client/pkg/v3/transport
pkg/cmd/readyz/readyz.go:15:2: cannot find package "." in:
	/Users/mustafa/workspace/go/src/etcd/cluster-etcd-operator/vendor/go.etcd.io/etcd/client/v3
make: *** [verify-govet] Error 1


```

go version : go1.16.15

Reply:
the client comes from a different package in 4.8, check the import paths:

https://github.com/openshift/cluster-etcd-operator/blob/release-4.8/pkg/etcdcli/etcdcli.go#L19

just check how the clientv3.Client is used in there and add the same stuff into the readiness command

Reply:
updated